build.gradle

```diff
@@ -610,6 +610,19 @@ project('spring-integration-mongodb') {
 	}
 }
 
+project('spring-integration-r2dbc') {
+	description = 'Spring Integration R2DBC Support'
+	dependencies {
+		api project(':spring-integration-core')
+		api ('org.springframework.data:spring-data-r2dbc') {
+			exclude group: 'org.springframework'
+		}
+		testCompile group: 'io.r2dbc', name: 'r2dbc-h2', version: '0.8.4.RELEASE'
```

Comment:
We need to extract `r2dbc-h2` version as a `r2dbch2Version` variable alongside with similar in this gradle config.